app/js/main.js

```diff
@@ -658,7 +675,31 @@ function _getDefaultDirectory() {
 }
 
 /**
-* COnverting frames to png
+ * Check the save directory still exists and recreate it if deleted.
+ */
+function checkDirectoryExists() {
+    "use strict";
+    // Check if save directory exists.
+    fs.stat(frameExportDirectory, function (err, stats) {
```

Comment:
`mkdirp` calls `fs.stat` and ensures it does not exist before creating, so doing it here creates unneeded overhead.